src/olympia/reviewers/models.py

```diff
@@ -1158,6 +1166,20 @@ def check_is_recommendable(cls, version):
             recommendable = item.recommendable
         return bool(recommendable)
 
+    @classmethod
+    def check_should_be_delayed(cls, version):
+        # 'Nominated' addons - add-ons that are waiting for approval of their
+        # first listed version - should have their auto-approval delayed for
+        # 24 hours to give us time to catch spam. Langpacks are exempted from
+        # this since they are submitted as part of Firefox release process.
+        is_langpack = version.addon.type == amo.ADDON_LPAPP
+        now = datetime.now()
+        nomination = version.nomination or version.addon.created
```

Comment:
I forget, is `version.nomination` inherited from any previous versions awaiting review?  We don't want a new upload on hour 23 to reset the clock again (I assume)

Reply:
It should be, yes (`Version.inherit_nomination()` called inside `from_upload()` and in `post_save`)